addons/mail/static/src/js/services/mail_manager.js

```diff
@@ -865,6 +865,15 @@ var MailManager =  AbstractService.extend({
         this._pinnedDmPartners = [];
         // all threads, including channels, DM, mailboxes, document threads, ...
         this._threads = [];
+
+        this._addMailbox({
+            id: 'inbox',
+            name: _t("Inbox"),
```

Comment:
translate in stable: need to add stuff in translation files but not sure exactly what